pulp_container/app/tasks/sync_stages.py

```diff
@@ -587,7 +594,7 @@ def _include_layer(self, layer):
         """
         foreign_excluded = not self.remote.include_foreign_layers
         layer_type = layer.get("mediaType", MEDIA_TYPE.REGULAR_BLOB)
-        is_foreign = layer_type in (MEDIA_TYPE.FOREIGN_BLOB, MEDIA_TYPE.FOREIGN_BLOB_OCI)
+        is_foreign = layer_type in (MEDIA_TYPE.FOREIGN_BLOB, MEDIA_TYPE.FOREIGN_BLOB_OCI_TAR_GZIP)
```

Comment:
i am afraid you will need to include here all the types of compression

Reply:
I appended only `MEDIA_TYPE.FOREIGN_BLOB_OCI_TAR_ZSTD`.